packages/template-retail-react-app/app/hooks/use-item-arrangement.js

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (c) 2025, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {useShopperBasketsMutation} from '@salesforce/commerce-sdk-react'
+import {useCallback} from 'react'
+import {DEFAULT_SHIPMENT_ID} from '@salesforce/retail-react-app/app/constants'
+
+/**
+ * Hook for item arrangement operations
+ * Focused only on moving items between shipments (basket data manipulation)
+ * @param {string} basketId - The basket ID
+ * @returns {Object} Object containing item arrangement functions
+ */
+export const useItemArrangement = (basketId) => {
```

Comment:
name is too vague, what do you think about `useItemShipmentManagement`, it focuses on the shipment management aspect